browser/extensions/api/brave_rewards_api.cc

```diff
@@ -1189,6 +1195,57 @@ BraveRewardsShouldShowOnboardingFunction::Run() {
   return RespondNow(OneArgument(base::Value(should_show)));
 }
 
+BraveRewardsGetScheduledCaptchaInfoFunction::
```

Comment:
I think ideally this would also go in a different api, but a follow-up is fine for that